stack.yaml

```diff
@@ -14,7 +14,7 @@ packages:
 - solver-benchmarks/
 
 extra-deps:
-- rere-0.2@sha256:ec1d87554c03755d89f3401c5e5c27827e61dde835c54cbdaf509136ce8f6f2b,4079
+- rere-0.2@rev:1
 
 allow-newer: true
```

Comment:
just curious, do you know if this is still required?

Reply:
yeah, i removed it and stack continues asking for. It is not stackage and i guess it will never be so 🤷 
will add a comment for future reference

Reply:
good question, will try without it

Reply:
```
[0mWARNING: Ignoring cabal-install's bounds on Win32 (>=2.8 && <3); using Win32-2.6.2.1.
Reason: allow-newer enabled.[0m
[0mWARNING: Ignoring hackage-security's bounds on Cabal (>=1.14 && <1.26 || >=2.0 && <2.6 || >=3.0 && <3.7); using Cabal-3.7.0.0.
Reason: allow-newer enabled.[0m
[0mWARNING: Ignoring hackage-security's bounds on Cabal-syntax (<3.7); using Cabal-syntax-3.7.0.0.
Reason: allow-newer enabled.[0m
[0mWARNING: Ignoring cabal-testsuite's bounds on Cabal (((==2.2.* || ==2.4.* || ==3.0.* || ==3.2.* || ==3.4.* || ==3.6.*) && ==3.7.0.0) && (==2.2.* || ==2.4.* || ==3.0.* || ==3.2.* || ==3.4.* || ==3.6.*)); using Cabal-3.7.0.0.
Reason: allow-newer enabled.[0m
[0mWARNING: Ignoring cabal-testsuite's bounds on Cabal-syntax ((==3.6.* && ==3.7.0.0) && ==3.6.*); using Cabal-syntax-3.7.0.0.
Reason: allow-newer enabled.[0m
```

Added comments